File: src/main/java/no/nav/fo/forenkletdeploy/unleash/streategies/IsDevStrategy.java
package no.nav.fo.forenkletdeploy.unleash.streategies;

import no.finn.unleash.strategy.Strategy;
import no.nav.fo.forenkletdeploy.service.UnleashService;

import java.util.Arrays;
import java.util.List;
import java.util.Map;

public class IsDevStrategy implements Strategy {
    @Override
    public String getName() {
        return "isDev";
    }

    @Override
    public boolean isEnabled(Map<String, String> parameters) {
        String env = UnleashService.getEnvironment();
        return env.equalsIgnoreCase("local");
    }
}

File: src/main/kotlin/no/nav/fo/forenkletdeploy/unleash/streategies/IsDevStrategy.kt
package no.nav.fo.forenkletdeploy.unleash.streategies

import no.finn.unleash.strategy.Strategy
import no.nav.fo.forenkletdeploy.util.Utils.getEnvironment

class IsDevStrategy : Strategy {
    override fun getName(): String {
        return "isDev"
    }

    override fun isEnabled(parameters: Map<String, String>): Boolean =
        getEnvironment().equals("local", ignoreCase = true)
}
